src/groups/bmq/bmqu/bmqu_blob.t.cpp

```diff
@@ -1090,7 +1090,7 @@ static void test14_readUpToNBytes()
         {L_, bmqu::BlobPosition(3, 0), 3, 3, "ghi"},
         {L_, bmqu::BlobPosition(3, 0), 12, 3, "ghi"},
         {L_, bmqu::BlobPosition(4, 0), 0, 0, ""},
-        {L_, bmqu::BlobPosition(4, 0), 1, -2, ""},  // invalid length
+        {L_, bmqu::BlobPosition(4, 0), 1, 0, ""},
```

Comment:
This is a valid case now.
It is okay to try to read `up to` 1 byte in the end of the blob and read 0 bytes.

Reply:
+ add comment